avocado_vt/loader.py

```diff
@@ -79,6 +79,28 @@ def arch_listing(config, guest_name_parser=None):
     LOG.debug("")
 
 
+def get_vt_test_tags(vt_params):
+    """
+    Returns the test categories based on a `tags` vt param
+
+    :rtype: dict
+    """
+    tags = vt_params.get('tags', '').strip()
+    result = {}
+    for tag in tags.split(','):
+        if not tag:
+            continue
+        if ':' in tag:
```

Comment:
Will we have the syntax and meaning of colon here documented somewhere in the project docs?

Reply:
It's given by [this doc](https://avocado-framework.readthedocs.io/en/85.0/guides/user/chapters/tags.html#using-further-categorization-with-keys-and-values), do you think we also need to highlight it in VT?

Reply:
Oh, then it is fine. I forgot this is well documented on the Avocado side, VT should be fine since it follows the same rules :+1: